web/themes/custom/votegov/templates/component/usa-identifier.html.twig

```diff
@@ -34,8 +34,13 @@
     <div class="usa-identifier__container">
       <p>
         {% set usagov_link = "Visit USA.gov" | t %}
+        {% if language != "es" %}
+          {% set usagov_url = "https://www.usa.gov/" %}
+        {% else %}
+          {% set usagov_url = "https://www.usa.gov/es" %}
+        {% endif %}
```

Comment:
i think we should swith the conditional to set the url if it is spanish so something like this

```suggestion
        {% if language == "es" %}
          {% set usagov_url = "https://www.usa.gov/es" %}
        {% else %}
          {% set usagov_url = "https://www.usa.gov" %}
        {% endif %}
```